acm/src/main/java/org/literacybridge/acm/gui/ResourceView/audioItems/ColumnInfo.java

```diff
@@ -0,0 +1,107 @@
+package org.literacybridge.acm.gui.ResourceView.audioItems;
```

Comment:
Please document what the class is for and how it is used, especially how different types of T are used.